ledger/src/blocktree_processor.rs

```diff
@@ -943,7 +943,7 @@ pub mod tests {
         assert_eq!(bank_forks_info.len(), 1);
         assert_eq!(bank_forks_info[0], BankForksInfo { bank_slot: 0 });
         let bank = bank_forks[0].clone();
-        assert_eq!(bank.tick_height(), 0);
+        assert_eq!(bank.tick_height(), 1);
```

Comment:
@mvines can you double check this? I'm not sure why `bank.tick_height()` was expected to be 0 after filling slot 0 with one tick

Reply:
The first tick in slot 0 used to not count toward tick height. This way the number of ticks in slot 0 would be the same as the other slots. 

For instance, if genesis_block.ticks_per_slot is equal to 10, then the zeroth slot will have bank.max_tick_height of 9, so PoH will, starting at 0, generate a tick for tick_heights 0->1, 1->2, 2->3...8->9, which is only 9 ticks. The next slot, slot 1, will then generate 10 ticks because it's max tick height is 19, and it will start generating a tick for tick heights 9->10, 10->11, ... 18 -> 19. This is all because in the past we wanted to keep the first tick in a slot as 0-indexed.

Reply:
Ah right, and so if there is only one tick per slot, the tick height should be 0. Cool, this change seems fine then. Just missed cleaning in up in my other PR: https://github.com/solana-labs/solana/pull/6263

Thanks @carllin!